rust-src/id/src/test.rs

```diff
@@ -102,15 +103,46 @@ pub fn test_create_aci<T: Rng>(csprng: &mut T) -> AccCredentialInfo<ArCurve> {
     }
 }
 
+/// Create random IdObjectUseData to be used by tests
+pub fn test_create_id_use_data<T: Rng>(csprng: &mut T) -> IdObjectUseData<IpPairing, ArCurve> {
+    let aci = test_create_aci(csprng);
+    let randomness = ps_sig::SigRetrievalRandomness::generate_non_zero(csprng);
+    IdObjectUseData { aci, randomness }
+}
+
 /// Create PreIdentityObject for an account holder to be used by tests,
 /// with the anonymity revocation using all the given ars_infos.
-pub fn test_create_pio<'a>(
+pub fn test_create_pio_old<'a>(
```

Comment:
Will this be removed? Otherwise it would probably be better to name it with `_v0` or some such.